source/blog/2016-09-21-running-tempest-on-rdo-openstack-newton.html.md

```diff
@@ -58,12 +58,27 @@ Roll out a vm of [CentOS 7](https://www.centos.org/download/), Follow these step
 	      python-neutron-tests python-cinder-tests python-nova-tests python-swift-tests \
 	      python-ceilometer-tests python-gnocchi-tests python-aodh-tests
 
+	OR you can automatically install the required tempest plugins of the configured services on the environment.
+
+	    # python /usr/share/openstack-tempest-*/tools/install_test_packages.py
+
 3. To find what are tempest plugins installed:
 
 	    # tempest list-plugins
 
    Once done, you are ready to run tempest.
 
+   If you face any entry point issues while running tempest, you can debug using [entry_point_inspector](https://pypi.python.org/pypi/entry_point_inspector) tool
+   Install entry_point_inspector from epel.repo.
+
+            # yum install epel-release
```

Comment:
Leaving EPEL enabled might introduce some issues later, we could either disable it after installing epi or rebuild it in CBS and ship in RDO common.